src/PointwiseFunctions/AnalyticData/GrMhd/MagnetizedTovStar.hpp

```diff
@@ -146,7 +148,8 @@ namespace grmhd::AnalyticData {
  * Note that the magnetic field strength goes as \f$A_b\f$ so any desired value
  * can be achieved by a linear scaling.
  */
-class MagnetizedTovStar : public MarkAsAnalyticData,
+class MagnetizedTovStar : public virtual evolution::initial_data::InitialData,
```

Comment:
Q. Is this line strictly required? (RelativisticEuler TovStar already inherits `InitialData`)

Reply:
Thank you!